Dockerfile_mbtci_java17

```diff
@@ -0,0 +1,90 @@
+FROM sapmachine:17
+
+# Build time arguments
+ARG MTA_USER_HOME=/home/mta
+ARG MBT_VERSION=1.2.8
+ARG GO_VERSION=1.15.6
```

Comment:
maybe it makes sense also to update Golang (at least to a highest bugfix version)
```suggestion
ARG GO_VERSION=1.15.15
```